.github/workflows/lighthouse.yml

```
@@ -0,0 +1,96 @@
+            http://localhost/dashboard
+            http://localhost/dashboard/map
+          uploadArtifacts: true
+          temporaryPublicStorage: true
+          extraHeaders: '{"Cookie": "token=${{ steps.login.outputs.token }}"}'
+
+      - name: Format lighthouse score
+        id: format_lighthouse_score
+        uses: actions/github-script@v7
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          script: |
```

actionlint (1.7.7)

property "manifest" is not defined in object type {assertionresults: string; links: string; resultspath: string}

(expression)